src/ChannelServer/Database/Account.cs

```diff
@@ -87,6 +88,26 @@ public void AddChatMacro(ChatMacro macro)
 		/// <returns></returns>
 		public ChatMacro[] GetChatMacros()
 		{
+			// Initialize with the default set of macros if none exist.
+			if (_chatMacros.Count() == 0)
```

Comment:
Worth it to reduce nesting?

Reply:
Would this make more sense in the ctor, so it doesn't has to be checked on every call?